server/storage.lua

```diff
@@ -6,7 +6,7 @@
 
 ---@param request InsertVehicleEntityRequest
 function InsertVehicleEntity(request)
-    MySQL.insert('INSERT INTO player_vehicles (license, citizenid, vehicle, hash, mods, plate, garage, state) VALUES (?, ?, ?, ?, ?, ?, ?, ?)', {
+    local vehicleId = MySQL.insert.await('INSERT INTO player_vehicles (license, citizenid, vehicle, hash, mods, plate, garage, state) VALUES (?, ?, ?, ?, ?, ?, ?, ?)', {
```

Comment:
Shouldn't we be calling qbx_vehicles API for this instead?

Reply:
I'll have to modify the API